lib/loadcase.m

```diff
@@ -50,6 +50,9 @@
 end
 
 %%-----  read data into struct  -----
+if isstring(casefile)
```

Comment:
```suggestion
if isa(casefile, 'string')
```

Reply:
Thanks Timon.

Looks like strings were introduced in Matlab R2017a. For Octave, as far as I can tell strings are still not implemented and `isstring()` was only added in version 4.4.

So, I think the suggested change here would take care of backward compatibility.

Reply:
And it would probably be good to update the error message to ...
```
input arg should be a struct or a char array/string containing a filename
```